nens_auth_client/backends.py

```diff
@@ -56,11 +60,20 @@ def authenticate(self, request, claims):
         Returns:
           user or None
         """
+        allow_username_match = int(claims.get("custom:from_sso", 0)) == 1
```

Comment:
Die `== 1` voelt wat krakkemikkig. `bool()` geeft duidelijker aan dat je de reguliere `0=>False, rest=>True` wil hebben:

```suggestion
        allow_username_match = bool(int(claims.get("custom:from_sso", 0)))
```

Reply:
Wat ik hier wil is 1=>True rest=>False , dat is iets anders! Ik denk dat het goed is om strict te zijn.

Reply:
ok